### ISSUE TITLE
Ava tests are errors on wallaby start

### ISSUE THREAD

**Issue by danzel** [wallabyjs/public#711]
### Issue description or question

I have a typescript project using ava for tests.
When I first open this project in VSCode and start wallabyjs (or restart wallabyjs while running) I get the following errors in the Wallaby.js Test console:

```
​TypeError: ava_1.default is not a function
​ c:\Users\dave\Code\cc2server\test\repositories\postgres\combinationRepositorySpec.ts:10:0
​TypeError: ava_1.default is not a function
​ c:\Users\dave\Code\cc2server\test\repositories\postgres\cookieRepositorySpec.ts:10:0
​TypeError: ava_1.default is not a function
​ c:\Users\dave\Code\cc2server\test\repositories\postgres\playerRepositorySpec.ts:10:0
​TypeError: ava_1.default is not a function
​ c:\Users\dave\Code\cc2server\test\repositories\postgres\teamRepositorySpec.ts:10:0
​TypeError: ava_1.default is not a function
​ c:\Users\dave\Code\cc2server\test\util\generatorSpec.ts:5:0
```

These correspond with the first test in each file
![image](https://cloud.githubusercontent.com/assets/393086/17200680/857bfbc2-54db-11e6-9d03-cc928aed1cd3.png)

Resulting in wallaby thinking I have no tests.
`​0 failing tests, 0 passing`.

If I edit and revert a file (so wallaby re-runs it) then that file will be fixed. Sometimes I can just touch the top/bottom a file. Sometimes I need to touch individual tests (wallaby being too smart for its own good!)

I can create a test project if required, but hopefully this is enough to reproduce and fix it :-)
Thanks!

### Wallaby.js configuration file

```javascript
module.exports = function (w) {

	return {
		files: [
			'migrations/*.sql',
			'src/**/*.ts',
			'test/utils/**/*.ts'
		],

		tests: [
			'test/**/*Spec.ts'
		],

		testFramework: 'ava',
		env: {
			type: 'node'
		},

		delays: {
			run: 500
		}
	};
};
```

### Code editor or IDE name and version

Visual Studio Code v1.3.1

### OS name and version

Windows

**Comment by ArtemGovorov** [comment]
Could you please paste your `tsconfig.json` and also specify how do you compile TS to run the code in prod (`ts-node`/precompiled/something else)?

**Comment by danzel** [comment]
tsconfig.json:
```json
{
	"compilerOptions": {
		"outDir": "./dist",
		"sourceMap": true,
		"noImplicitAny": true,
		"module": "commonjs",
		"target": "es6",
		"experimentalDecorators": true
	},
	"files": [
		"typings/index.d.ts",
		"src/server.ts",

		"test/repositories/postgres/combinationRepositorySpec.ts",
		"test/repositories/postgres/cookieRepositorySpec.ts",
		"test/repositories/postgres/playerRepositorySpec.ts",
		"test/repositories/postgres/teamRepositorySpec.ts",
		"test/util/generatorSpec.ts"
	]
}
```

I Haven't got as far as running the code outside of wallaby yet. The ava errors are only occurring in Spec files which I wouldn't compile for prod.
I have just successfully ran `tsc` which built the app (not tests)

**Comment by ArtemGovorov** [comment]
Thanks for the details. The issue is fixed in core v1.0.269. You may wait for a few minutes before it updates automatically, or [force the update now](https://wallabyjs.com/docs/intro/troubleshooting.html#forcing-wallabyjs-core-update).

**Comment by danzel** [comment]
Awesome, just confirming this is fixed. Thanks!
